### PR TITLE
Update failsafe.html

### PR DESCRIPTION
**Pull request by iamoverit**
fix for gps_rescue_descent_dist according to cli command
```
# get gps_rescue_descent_dist
gps_rescue_descent_dist = 20
Allowed range: 10 - 500
```

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by nerdCopter** [comment]
may need to set min based on MSP version. (did not investigate, only assuming -- i'll look this week).
via javascript, possibly jQuery.

**Comment by haslinghuis** [comment]
Thanks @iamoverit anyway :yum:

**Pull request closed without merging**
